frontend/src/components/FindChain/ChainNotFound.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import { Button, makeStyles, Typography } from "@material-ui/core";
+
+import { Close as CloseIcon } from "@mui/icons-material";
+
+import theme from "../../util/theme";
+
+export const ChainNotFound = ({
+  searchTerm,
+  backAction,
+}: {
+  searchTerm: string;
+  backAction: any;
+}) => {
+  const classes = makeStyles(theme as any)();
+
+  const { t } = useTranslation();
+  const history = useHistory();
+
+  return (
+    <div className={classes.alertContainer}>
+      <CloseIcon onClick={backAction} className={classes.closeIcon} />
+
+      <Typography component="h1">
+        {`${t("noLoopsFoundIn")}`} <span>{searchTerm}</span>
+      </Typography>
+
+      <Typography component="p">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
If you guys want to go live soon, then it might be a good idea to start removing all the placeholder text

Reply:
I will create an issue for this. I believe @giuliamml will know better for this

Reply:
We are still waiting for the content from the copywriter, so will take care of replacing all 'Lorem ipsum' once I get the right copy